src/js/flow/acquia_lift.variations.views.js

```diff
@@ -386,7 +381,7 @@
      */
     VariationTypeMenuListView: Backbone.View.extend({
       tagName: 'ul',
-      className: 'acquia-lift-page-variation-list',
+      className: 'acquia-lift-element-variation-list',
```

Comment:
Yes, the Qunit tests have showed up in my search for this one. To avoid extra work, I have recorded and delayed all test-related changes.

We'll see to remove or update them.